src/js/input-range/input-range.jsx

```diff
@@ -199,7 +204,7 @@ export default class InputRange extends React.Component {
     if (this.isMultiValue()) {
       return values.min >= this.props.minValue &&
              values.max <= this.props.maxValue &&
-             values.min < values.max;
+             (values.max - values.min) >= this.props.minRange;
```

Comment:
I think the current solution (`1` as the default `minRange` value) only works if step value is greater or equal to one. But it doesn't work if the step value is less than one.

I suggest making `minRange` optional with no default value. If `minRange` is not provided, use `step` to check against `max/min` values here. To ensure `minRange` is a multiple of `step`, add a check in `minimumRangePropType`.